agent/src/com/cloud/agent/Agent.java

```diff
@@ -140,9 +143,11 @@ public int value() {
     long _startupWait = _startupWaitDefault;
     boolean _reconnectAllowed = true;
     //For time sentitive task, e.g. PingTask
-    private final ThreadPoolExecutor _ugentTaskPool;
+    ThreadPoolExecutor _ugentTaskPool;
     ExecutorService _executor;
 
+    Thread _shutdownThread = new ShutdownThread(this);
```

Comment:
Would you mind not introducing `_` in the code anymore?
They are meaningless in our code base.

Reply:
I'm not fan of them either, but added the name to keep the naming consistent in the Agent class. Sure, will fix this for all the variables in Agent then.

Reply:
Awesome! Thanks!
This is a cosmetic thing, but if we do not get ride of them, newcomers might think that we enforce them. I am trying to do the same in other PRs.